src/main/java/it/pagopa/pn/paperchannel/middleware/db/dao/common/BaseDAO.java

```diff
@@ -62,28 +63,41 @@ protected BaseDAO(DynamoDbEnhancedAsyncClient dynamoDbEnhancedAsyncClient,
     }
 
     protected CompletableFuture<T> put(T entity){
+        log.logPuttingDynamoDBEntity(dynamoTable.tableName(), entity);
```

Comment:
questi log (in entrata e uscita) dovrebbero già essere eseguiti dal DynamoDBEnhancedAsyncClient esposto da pn-commons, non serve riperterli

Reply:
Lasciati attivi come da accordi